lib/mix/lib/mix/tasks/archive.install.ex

```diff
@@ -14,7 +14,7 @@ defmodule Mix.Tasks.Archive.Install do
 
   The argument can be an archive located at some URL:
 
-      mix archive.install http://example.com/foo.ez
+      mix archive.install https://example.com/foo.ez
```

Comment:
I think we can skip this change, it doesn't add anything to the documentation IMO 😕